jwst/datamodels/schemas/slitdata.schema.yaml

```diff
@@ -80,6 +80,18 @@ allOf:
       default: ""
       fits_keyword: SHUTSTA
       fits_hdu: SCI
+    quadrant:
+      title: MSA quadrant for slit
+      type: integer
+      fits_hdu: SCI
```

Comment:
Because these don't have a `fits_keyword` attribute, do they need to have `fits_hdu` defined?